cms/djangoapps/contentstore/management/commands/tests/test_delete_orphans.py

```diff
@@ -116,6 +116,6 @@ def create_split_course_with_published_orphan(self):
         # there should be one in published
         self.assertOrphanCount(course.id, 0)
         self.assertOrphanCount(published_branch, 1)
-        self.assertIn(orphan, self.store.get_items(published_branch))
+        self.assertIn(orphan.location, [x.location for x in self.store.get_items(published_branch)])
```

Comment:
Had to make two changes to the tests which depended on the existence of `__eq__`. One of them is this.